etna/search/tests/test_search_tags.py

```diff
@@ -61,43 +57,3 @@ def test_non_existant_parameter(self):
         result = query_string_exclude(context, "page", "1")
 
         self.assertEqual(result, "test=true")
-
-
-class GetSelectedFiltersTest(SimpleTestCase):
-    def setUp(self):
-        self.factory = RequestFactory()
-
-    def test_no_selected_filters_returns_empty_dict(self):
-        context = {"request": self.factory.get("?test=true&to-remove=1")}
-
-        selected_filters = get_selected_filters(context)
-
-        self.assertEqual(selected_filters, {})
-
-    def test_selected_collection(self):
-        context = {"request": self.factory.get("?collections=collection:MAF")}
-
-        selected_filters = get_selected_filters(context)
-
-        self.assertEqual(
-            selected_filters,
-            {
-                "collections": ["collection:MAF"],
-            },
-        )
-
-    def test_multiple_selected_filters_with_same_param(self):
-        context = {
-            "request": self.factory.get(
-                "?collections=collection:MAF&collections=collection:WO"
-            )
-        }
-
-        selected_filters = get_selected_filters(context)
-
-        self.assertEqual(
-            selected_filters,
-            {
-                "collections": ["collection:MAF", "collection:WO"],
-            },
-        )
```

Comment:
_class GetSelectedFiltersTest(SimpleTestCase):_ Why are these test cases removed?

Reply:
These are for `get_selected_filters` tag, which has been removed

Reply:
Ok